pkg/server/load_endpoint.go

```diff
@@ -50,6 +50,17 @@ type loadEndpoint struct {
 	mainMetricSource metricMarshaler
 }
 
+// customLoadEndpointMetricsSet returns the set of all custom metric names that
+// the load endpoint should export.
+func customLoadEndpointMetricsSet() map[string]struct{} {
```

Comment:
nit: the `custom` prefix doesn't really make sense to me. It makes me think of the custom metric page in dbconsole. If anything, in the context of the load endpoint, the custom metrics would be the metrics that are not exported as part of the /status/_vars endpoint. What do you think of changing the name to `loadEndpointMetrics`?